l/lico-kininike.md

```diff
@@ -0,0 +1,13 @@
+# Lico kininike
+
+![take a look at a pucture of lico-kininike](lico-kininike.md.JPG)
```

Comment:
You can use the Artist's name as the alt-text rather.
Please use `lico-kininike.jpg` as the filename for the image, and rename the image too.